README.md

```diff
@@ -32,11 +33,11 @@ a commercial set of Instagram-like filters.
 
 ## How it works
 
-This method is based on
+The accurate color reconstruction method is based on
```

Comment:
This looks like a method name, while it's not.

Reply:
"This method provides an accurate color reconstruction, and is based on"